### PR TITLE
Add contact links to issue template page

### PR DESCRIPTION
**Pull request by i509VCB**
This should help direct people who are asking for help to the discord or GitHub Discussions if it is not directly related to issues with or feature requests for Fabric API.

Screenshot (Outdated):
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: FabricMC Discord
+    url: https://discord.gg/v6v4pMv
+    about: Please ask here for help with installing Fabric
+  - name: Github Discussions
```
Comment (haykam821) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
  - name: GitHub Discussions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    url: https://github.com/FabricMC/fabric/discussions
+    about: Please look here for frequently asked questions and questions on how to use Fabric Api.
```
Comment (haykam821) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Please look here for frequently asked questions and questions on how to use Fabric API.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: FabricMC Discord
+    url: https://discord.gg/v6v4pMv
+    about: Please ask here for help with installing Fabric
```
Comment (haykam821) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Please ask here for help with installing Fabric.
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Review by modmuss50 — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by haykam821 — APPROVED** [review]

**Review by Player3324 — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
You should add the Fabric Wiki (https://fabricmc.net/wiki)

**Review by hsynlybasalbady-glitch — COMMENTED** [review]
https://fabricmc.net/use/installer

**Pull request merged**